arviz/plots/mcseplot.py

```diff
@@ -134,28 +135,28 @@ def plot_mcse(
     (figsize, ax_labelsize, titlesize, xt_labelsize, _linewidth, _markersize) = _scale_fig_size(
         figsize, textsize, rows, cols
     )
-    kwargs.setdefault("linestyle", kwargs.pop("ls", "none"))
-    kwargs.setdefault("linewidth", kwargs.pop("lw", _linewidth))
-    kwargs.setdefault("markersize", kwargs.pop("ms", _markersize))
+    kwargs.setdefault("linestyle", "none")
```

Comment:
`kwargs` should be dealiased before setting the defaults.

Reply:
I must have missed this one.